### PR TITLE
Feat/beslisboom mvp

### PR DESCRIPTION
**Pull request by bob-voorhoede**
- decision tree built with vue, using real data from Dato
- decision tree is now a modular content item
- data injected in njk template; picked up as prop by Vue
- special filters to transform Eleventy data into usable JS/Vue data
- added special flag for branches being an 'end result'

### CODE REVIEW

File: src/components/modular-content/decision-tree/decision-tree.njk [hunk 1]
```diff
@@ -1,9 +1,17 @@
+{% macro decisionTree(blockData) %}
+<div class="container">
+  <div id="decision-tree-app">
```
Comment (Hornebom) [inline src/components/modular-content/decision-tree/decision-tree.njk:3]: Can you make this id unique? what if we have multiple apps in one page

File: src/components/modular-content/decision-tree/decision-tree.njk [hunk 2]
```diff
@@ -18,16 +26,17 @@
         <p
           v-for="choice in option.choices"
           class="tree__answer"
-          :class="{ 'tree__answer--selected': option.selection === choice.pointsToSlug }"
-          @click="addChoice(option, choice.pointsToSlug)"
+          :class="{ 'tree__answer--selected': option.selection === choice.targetSlug }"
+          @click="addChoice(option, choice.targetSlug)"
         >
           {{ choice.content }}
         </p>
```
Comment (Hornebom) [inline src/components/modular-content/decision-tree/decision-tree.njk:33]: can you check for more semantic markup (if that's in scope of an mvp)

File: src/components/modular-content/decision-tree/decision-tree.scss [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+    }
   }
 
   &__question {
     border: thin solid #dbdbdb;
     padding: 1rem;
+    margin: 0;
   }
 
   &__answers {
```
Comment (Hornebom) [inline src/components/modular-content/decision-tree/decision-tree.scss:16]: I merged the new variables, so you can update from main and use them already

File: src/components/modular-content/index.njk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    {% set itemType = item._modelApiKey %}
+    {% set component = componentByType[itemType] %}
+    {% if component %}
+      {{ component(item, context) }}
+    {% else %}
+      <script>console.warn('Unknown item type "{{ itemType }}"');</script>
+    {% endif %}
+  {% endfor %}
+</div>
+{% endmacro %}
```
Comment (Hornebom) [inline src/components/modular-content/index.njk:22]: mooi dit

File: src/components/modular-content/rich-text/rich-text.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
+.rich-text {
+  a {
+    text-decoration: none;
+    color: clr(link);
```
Comment (Hornebom) [inline src/components/modular-content/rich-text/rich-text.scss]: hoe werkt dit?

File: src/components/modular-content/rich-text/rich-text.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  p {
+    margin-bottom: 26px;
```
Comment (Hornebom) [inline src/components/modular-content/rich-text/rich-text.scss:12]: set in relative `em`

Comment (Hornebom) [reply src/components/modular-content/rich-text/rich-text.scss:12]: and margin for all descendants
```css
  > * + * {
    margin-top: 1em;
  }
```

**Review by Hornebom — CHANGES_REQUESTED** [review]

File: src/components/modular-content/decision-tree/decision-tree.njk (hunk 1 above)
Comment (bob-voorhoede) [reply src/components/modular-content/decision-tree/decision-tree.njk:3]: not for mvp

File: src/components/modular-content/decision-tree/decision-tree.njk (hunk 2 above)
Comment (bob-voorhoede) [reply src/components/modular-content/decision-tree/decision-tree.njk:33]: i don't think it's in scope, but, what do you specifically mean with semantic markup in this case?

File: src/components/modular-content/decision-tree/decision-tree.scss (hunk 3 above)
Comment (bob-voorhoede) [reply src/components/modular-content/decision-tree/decision-tree.scss:16]: I think we can do this when we actually build it with real styles (#mvp)

File: src/components/modular-content/rich-text/rich-text.scss (hunk 6 above)
Comment (bob-voorhoede) [reply src/components/modular-content/rich-text/rich-text.scss:12]: pls do this when implementing real styles, this was just copied from other project #mvp

File: src/components/modular-content/rich-text/rich-text.scss (hunk 5 above)
Comment (bob-voorhoede) [reply src/components/modular-content/rich-text/rich-text.scss]: ah dat is een functie van Zowie - sloop ik er wel even uit

**Pull request merged**